src/app/static/src/app/index.ts

```diff
@@ -3,12 +3,19 @@ import {store} from "./main"
 import {router} from "./router";
 import UserHeader from "./components/header/UserHeader.vue";
 import Errors from "./components/Errors.vue";
+import Stepper from "./components/Stepper.vue";
+import Versions from "./components/versions/Versions.vue";
 import {mapActions, mapState} from "vuex";
 import {RootState} from "./root";
 import VueRouter from "vue-router";
 
 Vue.use(VueRouter);
 
+router.addRoutes( [
```

Comment:
The routes were originally being added in router.ts when router is created, but I had to move them out of that module as the merge with the ADR widget caused a circular dependency: Stepper -> ADR component -> Root Mutations -> router -> Stepper... 
Other ways of doing that, but this seemed easiest while not being too hacky.